internal/development.md

```
@@ -372,4 +372,14 @@ RAGFlow(user)> asr with 'FunAudioLLM/SenseVoiceSmall@test@siliconflow' audio './
 +----------------------------------------------------------------------------------------------------------------------+
 | The examination and testimony of the experts enabled the commission to conclude that five shots may have been fired. |
 +----------------------------------------------------------------------------------------------------------------------+
+```
+
+### 6.25 Optical Character Recognition\
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)